<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>701</width>
      <height>581</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(113, 113, 113);
color: rgb(255, 255, 255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
   
    font: bold 15px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ekran_kapa">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">    background-color: navy;
color: rgb(255, 255, 255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
   
    font: bold 15px;
</string>
    </property>
    <property name="text">
     <string>CLOSE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ekran_ac">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">    background-color: navy;
color: rgb(255, 255, 255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
   
    font: bold 15px;
</string>
    </property>
    <property name="text">
     <string>OPEN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="maske1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>490</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="maske2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>490</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="maske3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>490</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="maske5">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>490</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="maske6">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>490</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="maske7">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>490</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelmaske1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>500</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelmaske1_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>500</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelmaske1_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>500</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelmaske1_4">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>500</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelmaske1_5">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>500</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelmaske1_6">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>500</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>857</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
